package.json

```diff
@@ -15,7 +15,7 @@
   },
   "license": "MIT",
   "engines": {
-    "node": "^14.18.0 || ^16.13.0 || >=18.0.0"
+    "node": "^16.13.0 || >=18.0.0"
```

Comment:
Are you intentionally dropping support for node 14 here? And if so, why?

Reply:
Several dependencies dropped support for it, and it is out of LTS completely, not even in maintenance https://nodejs.dev/en/about/releases/